tgui/packages/tgui/interfaces/PreferencesMenu/jobs/jobs/scientist.ts

```diff
@@ -5,17 +5,6 @@ const Scientist: Job = {
   name: "Scientist",
   description: "Do experiments, perform research, feed the slimes, make bombs.",
   department: Science,
-  alt_titles: [
-    "Scientist",
-    "Circuitry Designer",
-    "Xenobiologist",
-    "Cytologist",
-    "Nanomachine Programmer",
-    "Plasma Researcher",
-    "Anomalist",
-    "Lab Technician",
-    "Theoretical Physicist",
```

Comment:
Surely some of these could stay or be replaced? "Lab Technician", and a more generic "Researcher"?

Reply:
Lab Technician makes sense to have stay in